.circleci/config.base.yml

```diff
@@ -485,13 +487,13 @@ jobs:
     working_directory: ~/repo
 
   amplify_migration_tests_v6:
-    environment:
-      AMPLIFY_PATH: /home/circleci/.npm-global/lib/node_modules/@aws-amplify/cli/bin/amplify
     parameters:
       os:
         type: executor
         default: l_large
     executor: << parameters.os >>
+    environment:
+      AMPLIFY_PATH: /home/circleci/.npm-global/lib/node_modules/@aws-amplify/cli/bin/amplify
```

Comment:
because AMPLIFY_PATH is also specified by the 'executor: <<..>>' step, we must provide it AFTER if we want to override the value. This is the case for migration tests.